rgwadmin/rgw.py

```diff
@@ -310,18 +310,42 @@ def get_user_bucket_quota(self, uid):
         '''Return the quota set on every bucket owned/created by a user'''
         return self.get_quota(uid=uid, quota_type='bucket')
 
-    def set_quota(self, uid, quota_type, max_size_kb=None, max_objects=None,
-                  enabled=None):
+    @staticmethod
+    def _quota(max_size_kb=None, max_objects=None, enabled=None):
+        quota = ''
+        if max_size_kb is not None:
+            quota += '&max-size-kb=%d' % max_size_kb
+        if max_objects is not None:
+            quota += '&max-objects=%d' % max_objects
+        if enabled is not None:
+            quota += '&enabled=%s' % str(enabled).lower()
+        return quota
+
+    def set_quota(self, *args, **kwargs):
+        from warnings import warn
+        warn("Deprecated in favor of set_user_quota.")
```

Comment:
Prefer `warn("Deprecated in favor of set_user_quota.", DeprecationWarning)`

Reply:
Good catch.